pkg/apis/serving/v1beta1/register.go

```diff
@@ -19,7 +19,6 @@ package v1beta1
 import (
```

Comment:
I'd kill v1beta1 completely.  I'd left alpha around for @julz  and DomainMapping, which seems imminent.

Reply:
sure